docs/guide/examples.rst

```diff
@@ -335,9 +335,6 @@ will compute a running average and standard deviation of input features (it can
   # To demonstrate loading
   del model, env
 
-  # Load the agent
-  model = PPO.load(log_dir + "ppo_halfcheetah")
```

Comment:
Model loading should stay here (as it was removed just few lines up).

Reply:
That also works, but then it doesn't load with the env,  which was also deleted few lines up.

Reply:
Ah, you are right, my bad! No need for changes here :)